README.md

```diff
@@ -1,42 +1,70 @@
-## Temporal VS Code extension
+<h1 align="center">
+  <br>
+    <img src="https://assets.temporal.io/w/vscode-icon.png" alt="Temporal" width="128">
+  <br>
+  Temporal for VS Code
+  <br>
+  <br>
+</h1>
 
-This repo contains the code for the Temporal VS Code extension.
+<h4 align="center">Debug TypeScript workflows by their ID or history file. Set breakpoints in code or on history events.</h4>
 
-The extension's purpose is to help debug workflows and reduce the amount of boilerplate required to start a workflow replayer.
+## Usage
 
-The extension exposes a single command: "Temporal: Open Panel" that (as the name suggests) opens a WebView panel.
-The panel consists of these three views:
+- Install [the extension](https://marketplace.visualstudio.com/items?itemName=temporal-technologies.temporalio)
+- Add a file at `src/debug-replayer.ts` (or can [configure](#entrypoint) an alternate location):
```

Comment:
Just stress that this needs to be in the source directory.

Reply:
The extension isn't published yet, right?

Reply:
not yet. 

what do you mean by source dir? a location in the tsconfig's "include" or "files" fields? does this only apply to .ts files?